docs/audit_report/src/side_channels/02_04_sphincsplus.rst

```diff
@@ -0,0 +1,196 @@
+""""""""
+SPHINCS+
+""""""""
+
+Analysierte Variante:
+
+- SphincsPlus-sha2-128s-r3.1, deterministic & randomized
+
+Für die Analyse von SPHINCS+ wurde ein Hilfsprogramm geschrieben, das ähnlich wie das Botan CLI die zu analysierenden Funktionen aufruft.
+Für die Erzeugung der Signatur wird folgender Aufruf verwendet:
+
+.. code-block:: cpp
+
+    auto params = Botan::Sphincs_Parameters::create("SphincsPlus-sha2-128s-r3.1");
+    Botan::SphincsPlus_PrivateKey priv_key(params.algorithm_identifier(), priv_key_bits);
+
+    #if DETERMINISTIC
+        Botan::PK_Signer sig(priv_key, rng, "Deterministic");
+    #else
+        Botan::PK_Signer sig(priv_key, rng, "Randomized");
+    #endif
+        signature = sig.sign_message(message, rng);
+
+Der SPHINCS+ Standard verwendet innerhalb des Hypertrees eine Variante von XMSS und bezeichnet diese verkürzt als XMSS.
+Dem Standard folgend wird dieses Verfahren auch in diesem Abschnitt als XMSS bezeichnet.
+
+**Modifikation des Hypertrees**
+
+Die Laufzeit zur Erzeugung einer Signatur hängt von der Höhe des Hypertrees und der Anzahl der Merkle-Bäume ab.
+Um die Laufzeit zu reduzieren, wird die Gesamthöhe auf vier beschränkt und es werden zwei Merkle-Bäume mit jeweils einer Höhe von zwei verwendet.
+Diese Änderung hat keinen Einfluss auf die Code-Coverage der durchgeführten Seitenkanalanalyse.
+
+**Leakage Zusammenfassung**
+
+DATA detektiert keine Leaks in der analysierten SPHINCS+ Implementierung.
+Es werden lediglich Unterschiede in der Programmausführung in Phase 1 von DATA gefunden.
+In Phase 2 kann jedoch keine statistische Abhängigkeit zu den verwendeten, geheimen kryptografischen Schlüsseln nachgewiesen werden.
+
+Als Hintergrundinformation sind in den folgenden Abschnitten die Gründe für die Ausführungsunterschiede erläutert.
+Insgesamt zeigt die Analyse fünf Unterschiede in der Ausführung auf.
+Ein Unterschied wurde innerhalb des FORS-Verfahrens gefunden, die restlichen vier Unterschiede betreffen das XMSS-Verfahren.
+
+**Ausführungsunterschied: FORS - treehash**
+
+Das FORS-Verfahren basiert auf der Verwendung von Merkle-Bäumen.
+Der öffentliche FORS-Schlüssel ist der Wurzelknoten in einem Merkle-Baum.
+Die Kinder dieses Knotens sind Wurzelknoten einzelner Merkle-Bäume, die zur Signatur einer Nachricht verwendet werden.
+
+Die `treehash` Routine berechnet mithilfe der Blätter eines Merkle-Baums die darüber liegenden Knoten.
+Bestandteile einer FORS-Signatur sind die sogenannten Authentisierungsdaten.
+Das sind Knoten, die bei der Signaturerzeugung generiert werden und die für die Verifikation benötigt werden, um die jeweilige Wurzel des Merkle-Baums zu berechnen.
+Während der Verifikation werden Teile der Blätter abhängig von der Nachricht und der Signatur berechnet.
+Die restlichen, benötigten Knoten zur Berechnung der Wurzel sind die sogenannten Authentisierungsdaten, welche auch in der Signatur enthalten sind.
```

Comment:
Das ist eine inhaltliche Dopplung mit Zeile 51, oder?

Reply:
Fix in https://github.com/sehlen-bsi/botan-docs/pull/113/commits/d425b5eae174c553fcfb606d31c29898f8d900ce